config/config.go

```diff
@@ -38,25 +40,21 @@ const DefaultNamespace = "default"
 
 const ResultsDatastreamIndexPrefix = "logs-cloud_security_posture.findings"
 
-const (
-	InputTypeVanillaK8s = "cloudbeat/cis_k8s"
-	InputTypeEks        = "cloudbeat/cis_eks"
-	InputTypeAws        = "cloudbeat/cis_aws"
-)
+var ErrBenchmarkNotSupported = errors.New("benchmark is not supported")
 
 type Fetcher struct {
 	Name string `config:"name"` // Name of the fetcher
 }
 
 type Config struct {
-	Type       string                  `config:"type"`
 	AWSConfig  aws.ConfigAWS           `config:",inline"`
 	RuntimeCfg *RuntimeConfig          `config:"runtime_cfg"`
 	Fetchers   []*config.C             `config:"fetchers"`
 	KubeConfig string                  `config:"kube_config"`
 	Period     time.Duration           `config:"period"`
 	Processors processors.PluginConfig `config:"processors"`
 	BundlePath string                  `config:"bundle_path"`
+	Benchmark  string                  `config:"config.v1.benchmark"`
```

Comment:
This change requires a matching change to the integration, isn't it?

Reply:
We already have it https://github.com/elastic/integrations/pull/4752/files#diff-721297d41d05ea41cf84e1ee80fdda5ac230452caef26678a37e752e43fbb466R11